sound/soc/sof/ipc4-topology.c

```diff
@@ -2828,7 +2828,7 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 		msg->primary |= SOF_IPC4_MOD_INSTANCE(swidget->instance_id);
 
 		msg->extension &= ~SOF_IPC4_MOD_EXT_PARAM_SIZE_MASK;
-		msg->extension |= ipc_size >> 2;
+		msg->extension |= SOF_IPC4_MOD_EXT_PARAM_SIZE(ipc_size >> 2);
```

Comment:
this has no effect and neither does this fix anything right? Can we add that in the commit message?

Reply:
OK, it is just clarifying the sloppy implementation, no functional change.

Reply:
@ujfalusi ping.